lib/iris/cube.py

```diff
@@ -1117,19 +1117,36 @@ def remove_coord(self, coord):
         for factory in self.aux_factories:
             factory.update(coord)
 
-    def remove_cell_measure(self, cell_measure):
+    def remove_cell_measure(self, name_or_cell_measure):
         """
         Removes a cell measure from the cube.
 
         Args:
 
-        * cell_measure (CellMeasure)
-            The CellMeasure to remove from the cube.
+        * name_or_cell_measure
+            The cell measure to remove from the cube. As either
 
-        See also
-        :meth:`Cube.add_cell_measure()<iris.cube.Cube.add_cell_measure>`
+            (a) a :attr:`standard_name`, :attr:`long_name`, or
+            :attr:`var_name`. Defaults to value of `default`
+            (which itself defaults to `unknown`) as defined in
+            :class:`iris._cube_coord_common.CFVariableMixin`.
```

Comment:
@alastair-gemmell A lovely piece of clarification, which explains that `cell_measure` can be an actual `CellMeasure` instance or an appropriate string name :smile: